From 511eeae4ec0cdafa79c79cd5ce027545d3d070f9 Mon Sep 17 00:00:00 2001
From: cmotc <cmotc@openmailbox.org>
Date: Wed, 28 Jan 2015 12:59:05 -0500
Subject: [PATCH] added msm7k includes back into android.mk file

---
 media/libstagefright/Android.mk | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/media/libstagefright/Android.mk b/media/libstagefright/Android.mk
index fad11b4..f4175b3 100644
--- a/media/libstagefright/Android.mk
+++ b/media/libstagefright/Android.mk
@@ -97,6 +97,18 @@ endif
 
 ifneq ($(filter caf bfam legacy,$(TARGET_QCOM_AUDIO_VARIANT)),)
     ifeq ($(BOARD_USES_LEGACY_ALSA_AUDIO),true)
+        ifeq ($(call is-chipset-in-board-platform,msm7x27),true)
+            LOCAL_SRC_FILES += LPAPlayer.cpp
+            LOCAL_CFLAGS += -DLEGACY_LPA -DUSE_LPA_MODE
+        endif
+        ifeq ($(call is-chipset-in-board-platform,msm7x27a),true)
+            LOCAL_SRC_FILES += LPAPlayer.cpp
+            LOCAL_CFLAGS += -DLEGACY_LPA -DUSE_LPA_MODE
+        endif
+        ifeq ($(call is-chipset-in-board-platform,msm7x30),true)
+            LOCAL_SRC_FILES += LPAPlayer.cpp
+            LOCAL_CFLAGS += -DLEGACY_LPA -DUSE_LPA_MODE
+        endif
         ifeq ($(call is-chipset-in-board-platform,msm8960),true)
             LOCAL_SRC_FILES += LPAPlayerALSA.cpp TunnelPlayer.cpp
             LOCAL_CFLAGS += -DUSE_TUNNEL_MODE
-- 
2.1.4

